test/uuid.js

```diff
@@ -6,4 +6,8 @@ describe('UUID', function () {
     assert.equal(36, uuid.length);
     assert.equal('4', uuid.substring(14, 15));
   });
+
+  it('should generate a different UUID-4', () => {
+    assert.notEqual(UUID(), UUID());
```

Comment:
this test is helpful, but it's not super helpful for detecting collisions

would it be reasonable to loop 1000 times for example and ensure all 1000 are unique?

Reply:
fyi 1000 is just an example number.